ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockMiner.java

```diff
@@ -177,13 +177,9 @@ private void logProducedBlock(final Block block, final BlockCreationTiming block
             block.getHeader().getGasUsed(),
             (block.getHeader().getGasUsed() * 100.0) / block.getHeader().getGasLimit(),
             block.getHash(),
-            blockCreationTiming.end("log").getSeconds() / 1000.0);
+            blockCreationTiming.end("log").toMillis() / 1000.0));
 
-    if (LOG.isDebugEnabled()) {
-      log = log + ", timing [" + blockCreationTiming + "]";
-    }
-
-    LOG.info(log);
+    LOG.debug("Timing #{} / {}", block.getHeader().getNumber(), blockCreationTiming);
```

Comment:
You don't test here if Debug log are enabled ?

Reply:
in this case there is no need, because, if debug is not enabled there is no extra wasted computation done, since `blockCreationTiming.toString()` is not called, and `block.getHeader().getNumber()` is just accessing a var

Reply:
Yes, that's right, but if someone change it in the future without paying attention to the attributes computation, it will generate an overhead

Reply:
It is not blocking though.